src/FsYacc.Core/fsyaccast.fs

```diff
@@ -15,30 +15,30 @@ open FSharp.Text.Lexing
 let (|KeyValue|) (kvp:KeyValuePair<_,_>) = kvp.Key,kvp.Value
 
 
-type Identifier = string
-type Code = string * Position
+type Identifier = string * Range
```

Comment:
similarly to fslex, fsyacc's own AST needed to be augmented with ranges for primitive productions like identifiers, code blocks, terminals, and nonterminals. This likewise flowed up through the remainder of the AST shapes.